File: agrario-front/src/app/modelos/Cliente.ts
export class Cliente {
  constructor(
    public idCliente: number | null,  // <-- aquí el cambio
    public nombre: string,
    public tipoCliente: string,
    public documentoIdentidad: number,
    public telefono: string,
    public email: string,
    public codigoCliente: string
  ) {}
}

File: Agrario/src/main/java/pe/proyecto/agrario/agrario/modelo/Cliente.java
package pe.proyecto.agrario.agrario.modelo;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;



@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Table(name = "cliente")
public class Cliente {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id_cliente")
    private Long idCliente;

    private String nombre;
    private String tipoCliente;
    private Long documentoIdentidad;
    private String telefono;
    private String email;
    private String codigoCliente;
}
